src/SSA.jl

```diff
@@ -19,11 +19,11 @@ end
 - **alg** : the algorithm used (`Symbol`, either `:gillespie`, `jensen`, or `tjc`).
 - **tvc** : whether rates are time varying.
 "
-struct SSAArgs
-    x0::Vector{Int64}
-    F::Any
-    nu::Matrix{Int64}
-    parms::Vector{Float64}
+struct SSAArgs{X,Ftype,N,P}
+    x0::X
+    F::Ftype
```

Comment:
Note that this forces the specializing on `F`, so the performance consideration was removed from the README